tests/interactive/test_app.py

```diff
@@ -76,169 +74,3 @@ async def test_inputs():
 
         assert isinstance(app.current_module, ModuleOp)
         assert app.current_module.is_structurally_equivalent(expected_module)
-
```

Comment:
feels like a shame to remove the test entirely. Let's keep as much of the functionality covered as makes sense. For example, we can still test that the outputs are updated in the way that we expect when the model object changes, even if we don't test the user action for now.